step.yml

```diff
@@ -211,6 +211,19 @@ inputs:
     - "yes"
     - "no"
 
+- test_device_list_path:
+  opts:
+    category: Automatic code signing
+    title: Path of file containing the devices to be registered
+    summary: If this input is set, the Step will register the listed devices from this file with the Apple Developer Portal.
+    description: |-
+      If this input is set, the Step will register the listed devices from this file with the Apple Developer Portal.
+
+      The format of the file is a comma separated list of the identifiers. For example:
+      `00000000–0000000000000001,00000000–0000000000000002,00000000–0000000000000003`
+
+      Note that setting this will have a higher priority than the Bitrise provided devices list.
```

Comment:
Might we could mention what name we will use for registering the device on apple developer portal, so that users can search for bitrise registered devices easier?